src/porepy/applications/verification_setups/verification_utils.py

```diff
@@ -19,6 +19,54 @@ class VerificationUtils:
 
     """
 
+    mdg: pp.MixedDimensionalGrid
+    """Mixed-dimensional grid."""
+
+    stress_keyword: str
+    """Keyword used for accessing the parameters of the mechanical subproblem."""
+
+    bc_values_mechanics_key: str
+    """Keyword used for accessing the mechanical boundary values."""
+
+    def displacement_trace(
+        self, displacement: np.ndarray, pressure: np.ndarray
+    ) -> np.ndarray:
+        """Project the displacement vector onto the faces.
+
+        Parameters:
+            displacement: displacement solution of shape (sd.dim * sd.num_cells, ).
+            pressure: pressure solution of shape (sd.num_cells, ).
+
+        Returns:
+            Trace of the displacement with shape (sd.dim * sd.num_faces, ).
+
+        Raises:
+             Exception if the mixed-dimensional grid contains more that one subdomain.
```

Comment:
```suggestion:
Exception if the mixed-dimensional grid contains more than one subdomain.
```

Reply:
I don't understand what is the suggested change here.